DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -115,7 +115,7 @@ public async Task<Variable<T>> VariableAsync<T>(User user, string key, T default
             return variable;
         }
 
-        public async Task<Dictionary<string, Variable<object>>> AllVariablesAsync(User user)
+        public async Task<Dictionary<string, Dictionary<string, object>>> AllVariablesAsync(User user)
```

Comment:
this shouldn't be a dictionary mapping to `object`, it should be mapping to a known object shape since we know what all the fields are here. You should define some kind of `AllVariablesVariable` class or interface to use here